### PR TITLE
Event final visit confirmation + refactoring

### LINKED ISSUE #127: Event visit final confirmation
Send email to all approved visitors with request to final confirmitaion

### PR DESCRIPTION
**Pull request by sveredyuk**
@all Check how I refactored business logic in the model to policy.

I know that policy often use only for authorization, but I like also use it like knowledge and resolve store.

Fixed #127

### CODE REVIEW

**Review by kushniryb — APPROVED** [review]
Nice, I like it. Well done! :+1:

File: README.md [hunk 1]
```diff
@@ -25,6 +25,33 @@ We have special `#pivorak-web-app` channel for contributors.
...
+Model should containe only data-logic.
+- Processing -> to service/interactor.
+- Complex queries -> to finders
+- Predicates -> to policy.
+
+### View
+View should be clean as much as possible. Please, use helpers to avoid call methods with some params at template.
+
+### Controller
+Skinny controller. Only REST. Only 7 allowed actions (index, show, new, create, edit, update, destroy). If you need additional action - extract to new sub-constroller
```
Comment (denys-medynskyi) [inline README.md]: typo in word `constroller`

File: README.md [hunk 2]
```diff
@@ -25,6 +25,33 @@ We have special `#pivorak-web-app` channel for contributors.
...
+
+
+### Service
+Single responsibility resource action that represent businees logic.
```
Comment (denys-medynskyi) [inline README.md]: `represent businees` -> `represents business`

**Review by denys-medynskyi — APPROVED** [review]
fix typos if you have time

**Pull request merged**